_posts/2018-01-08-top-5-reasons-for-ruby-ists-to-use-crystal.md

```diff
@@ -1,185 +1,185 @@
----

```

Comment:
Why is there changes here?

Reply:
Replaced CRLF with LF. It's an independent commit but I didn't feel like opening another PR for this.